# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../isakmpd/lib64-libs.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- isakmpd-20041012.orig/sysdep/linux/GNUmakefile.sysdep.vanilla	2006-03-04 16:16:40.386863750 +0100
+++ isakmpd-20041012.orig/sysdep/linux/GNUmakefile.sysdep	2006-03-04 16:17:45.046904750 +0100
@@ -25,13 +25,11 @@
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
-LIBGMP:=	/usr/lib/libgmp.a
-LIBCRYPTO:=	/usr/lib/libcrypto.a
 LIBSYSDEPDIR:=	${.CURDIR}/sysdep/common/libsysdep
 LIBSYSDEP:=	${LIBSYSDEPDIR}/libsysdep.a
 
-LDADD+=		-lgmp ${LIBSYSDEP} ${LIBCRYPTO}
-DPADD+=		${LIBGMP} ${LIBSYSDEP}
+LDADD+=		-lcrypto -lgmp ${LIBSYSDEP} ${LIBCRYPTO}
+DPADD+=		-lcrypto -lgmp ${LIBSYSDEP}
 
 CFLAGS+=	-DUSE_OLD_SOCKADDR -DHAVE_PCAP \
 		-DNEED_SYSDEP_APP -DMP_FLAVOUR=MP_FLAVOUR_GMP \
